algorithms/hhl/hhl/hhl.ipynb

```diff
@@ -1,982 +1,1292 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.        if np.linalg.norm(sol_classical - qsol_corrected) / np.linalg.norm(sol_classical) &gt; 0.1 and check:
</pre><p>Since there might be ambiguity in the sign, there might be some false-positive here. I suggest to take only the absolute values, or maybe align the signs of both solutions (e.g., by <code>state_corrected=np.sign(state_corrected[0])*np.sign(state_classical[0])*state_corrected</code>)  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Classiq/classiq-library/pull/194/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Added sign ambiguity correction to function <code>quantum_solution_preprocessed</code>.</p>